functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -102,6 +113,11 @@ var _ = AfterSuite(func() {
 func Test5LatencyTesting(t *testing.T) {
 	RegisterFailHandler(Fail)
 
+	if !isTestExecutableFound() {
+		testlog.Infof("The test executable file %q does not exist, skipping the suite.", testExecutablePath)
+		t.Skip()
```

Comment:
continue the discussion from here: https://github.com/openshift-kni/performance-addon-operators/pull/930#discussion_r951280557 

If the make target failed to create the binary or the binary is missing for any other reason, what benefit do we have by skipping and not failing the test?
As you mentioned in the PR title - the suite works properly only if the binary exists.

Reply:
the tests run on top of the binary. It is a precondition for this suite to have the binary exists, otherwise there is no point in running the tests. we skip and don't fail the tests because it is not visible to the tests why the binary is not found. OTOH running this suite is done by make target that includes generating the binary:
https://github.com/openshift-kni/performance-addon-operators/blob/master/Makefile#L271

 running the suite in another way is not considered a valid run. In case of failure to create the binary that will be reflected upon running the make target that creates the binary which runs this file:
https://github.com/openshift-kni/performance-addon-operators/blob/master/hack/build-latency-test-bin.sh

Reply:
it is like requesting to have a PP on the cluster before running the functional tests, otherwise there is no point to run the suites. the reason for failing to create the PP should not be handled in each suite as we know this should be handled at earlier stage before getting to the phase of running the suite.

Reply:
Basically, if the compilation failed here:
https://github.com/openshift-kni/performance-addon-operators/blob/master/hack/build-latency-test-bin.sh#L10 

The make target that depends on it would be failed as well.

Reply:
then target commands will not be executed if the prerequisites of the target fail, so in this case, the suite will not even run.

Reply:
Exactly so there is no chance for the suite to start without the executable

Reply:
So why would we need the suite to skip in case the executable is missing?

Reply:
https://github.com/openshift-kni/performance-addon-operators/pull/938#issuecomment-1225507397